azure/packages/azure-client/README.md

```diff
@@ -65,6 +65,10 @@ const clientProps = {
 const azureClient = new AzureClient(clientProps);
 ```
 
+### Experimental Features
+
+`AzureClient` supports the ability to instantiate with experimental features enabled. These features are experimental in nauture and are **NOT** reccomended to be used in production scenarios. To learn more, see [Experimental Features](https://fluidframework.com/docs/build/experimental-features/).
```

Comment:
Is this functionality also supported in tinylicious-client / fluid-static? If so, it would probably be worth documenting in their READMEs too.

Reply:
Looks like we don't support feature gates in tinylicious-client. @ssimic2, should we expose feature gates there as well?

As per fluid-static, I don't think it applies since it doesn't access the loader level, but let me know if this sounds incorrect.